tasks/check-and-update.yml

```diff
@@ -13,6 +13,13 @@
   when: dig_result.stdout | length > 0
   check_mode: false
 
+- name: Delete ssh host keys
+  file:
+    path: "{{ item }}"
+    state: absent
+  with_fileglob:
+    - "/etc/ssh/ssh_host*"
+
```

Comment:
I think we need something like `when: template_vm|bool` at the end, else everytime this playbook is run against a non-template VM the ssh host keys get deleted.